ctapipe/containers.py

```diff
@@ -1368,43 +1289,88 @@ class SimulatedShowerDistribution(Container):
     )
 
 
-class ArrayEventContainer(Container):
-    """Top-level container for all event information"""
+class TelescopeEventContainer(Container):
+    """Container for single-telescope data"""
 
     index = Field(
-        default_factory=EventIndexContainer, description="event indexing information"
+        default_factory=TelescopeEventIndexContainer,
+        description="event indexing information",
     )
-    r0 = Field(default_factory=R0Container, description="Raw Data")
-    r1 = Field(default_factory=R1Container, description="R1 Calibrated Data")
+
+    simulation = Field(
+        None,
+        description="Simulated Event Information",
+        type=SimulationTelescopeContainer,
+    )
+
+    r0 = Field(default_factory=R0TelescopeContainer, description="Raw Data")
+    r1 = Field(default_factory=R1TelescopeContainer, description="R1 Calibrated Data")
     dl0 = Field(
-        default_factory=DL0Container, description="DL0 Data Volume Reduced Data"
+        default_factory=DL0TelescopeContainer,
+        description="DL0 Data Volume Reduced Data",
     )
-    dl1 = Field(default_factory=DL1Container, description="DL1 Calibrated image")
-    dl2 = Field(default_factory=DL2Container, description="DL2 reconstruction info")
-    simulation = Field(
-        None, description="Simulated Event Information", type=SimulatedEventContainer
+    dl1 = Field(
+        default_factory=DL1TelescopeContainer, description="DL1 images and parameters"
     )
-    trigger = Field(
-        default_factory=TriggerContainer, description="central trigger information"
+    dl2 = Field(
+        default_factory=DL2TelescopeContainer,
```

Comment:
@Tobychev here it is used. If it is missing in `__all__` I'll add it

Reply:
Yeah, looks to be missing from `__all__` to me.